wfe2/wfe.go

```diff
@@ -881,7 +892,7 @@ func (wfe *WebFrontEndImpl) prepChallengeForDisplay(request *http.Request, authz
 
 	// If the authz has been marked invalid, consider all challenges on that authz
 	// to be invalid as well.
-	if authz.Status == core.StatusInvalid {
+	if features.Enabled(features.ForceConsistentStatus) && authz.Status == core.StatusInvalid {
```

Comment:
Looks like this got re-added by accident; bad merge?

Reply:
Oops, good catch. Fixed in 2c45807